oci/runtime_oci.go

```diff
@@ -154,7 +154,9 @@ func (r *runtimeOCI) CreateContainer(c *Container, cgroupParent string) (err err
 	someData := []byte{0}
 	_, err = parentStartPipe.Write(someData)
 	if err != nil {
-		cmd.Wait()
+		if err := cmd.Wait(); err != nil {
+			return err
```

Comment:
Can we wrap the two errors? The `Write()` error may contain useful information.

Reply:
Yes, sure. Adapted as suggested.